etna/pipeline/base.py

```diff
@@ -124,13 +132,279 @@ def _validate_quantiles(quantiles: Sequence[float]) -> Sequence[float]:
                 raise ValueError("Quantile should be a number from (0,1).")
         return quantiles
 
-    def check_support_prediction_interval(self, prediction_interval_option: bool = False):
-        """Check if pipeline supports prediction intervals, if not, warns a user.
+    @abstractmethod
+    def _forecast(self) -> TSDataset:
+        """Make predictions."""
+        pass
```

Comment:
It should have be
```suggestion
        raise NotImplementedError
```

Reply:
We should change it every where then, I will create an issue